src/components/Comments.jsx

```diff
@@ -1,3 +1,35 @@
+import { getReviewComments } from "../api";
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+
 export const Comments = () => {
-  return <p>Comments Go Here</p>;
+  const [comments, setComments] = useState([]);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    getReviewComments(review_id).then((comments) => {
+      setComments(comments);
```

Comment:
Please use a 'loading' state to give a visual feedback of loading comments to the user.